install-on-azure/mount_disk.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+PUBLIC_NFS_SERVER_IP=$(jq -r '.HEADNODE_IP | .[0]' config.json)
+USERNAME=$admin_name
+if [ -n "$custom_shared_dir" ]; then
+    shared_dir=$custom_shared_dir
+    setup_nfs=false
+    echo "Using shared drive: $shared_dir"
+else
+    mount_point="/home/$USERNAME/neuraldb_enterprise"
```

Comment:
Can this be /opt/neuraldb_enterprise/model_bazaar? Just wondering since the database files are in /opt/neuraldb_enterprise/database, so it would be nice if they could be in the same place. If it's not possible due to mounting stuff or if you think this convention is better, let me know.